packages/expo-sqlite/src/Storage.ts

```diff
@@ -182,14 +182,14 @@ export class SQLiteStorage {
    * Alias for [`setItemAsync()`](#setitemasynckey-value).
    */
   async setItem(key: string, value: string | SQLiteStorageSetItemUpdateFunction): Promise<void> {
-    this.setItemAsync(key, value);
+    await this.setItemAsync(key, value);
```

Comment:
```suggestion
    return this.setItemAsync(key, value);
```
why not return the promise to the caller? Here we are awaiting a result and wrapping it in a new promise.